### ISSUE TITLE
Missing throttle controls from Sega Y-board games

### ISSUE THREAD

**Issue by PlagueWizard** [libretro/FBNeo#24]
All revisions of Galaxy Force II and G-LOC for the Sega Y-board lack throttle control and it does not appear to be bindable in RetroArch. This makes both of these games _much_ more difficult to the point of near impossibility in Galaxy Force II's case.

I went and checked and throttle controls were also missing from the last revision of Final Burn Alpha as well, but _not_ from the standalone emulator.

**Comment by barbudreadmon** [comment]
Indeed, Throttle isn't hooked, standalone is mapping it to "z-axis", it might be a good idea to map every z-axis controls to the right analog stick

**Comment by barbudreadmon** [comment]
Fixed in https://github.com/libretro/FBNeo/commit/cac418814359f02f5fad8b20be5a928e1838c78b
Closing, let me know if the issue persist.

**Comment by PlagueWizard** [comment]
Wow that was fast, thanks.

I just tried it, and while the throttle controls are now hooked, it seems to be overriding the y-axis controls and setting it to whatever you have the throttle hooked as.

**Comment by barbudreadmon** [comment]
Oops, fixed in https://github.com/libretro/FBNeo/commit/490d1fb537293c53b224b2f63944b4325bce1226.
I'll be honest though, while i see the control is definitely doing something (there is an animation on the boosters in gforce2), i don't really feel the effect, there might be something i don't understand about those controls.

**Comment by barbudreadmon** [comment]
Actually i feel the effect after merging https://github.com/libretro/FBNeo/pull/34/commits/c6bcd1ed52d0fca8d09cecff626a1b3db15542ff from upstream so i guess the control wasn't working properly. Closing again, next step would be to use the right analog trigger instead of right analog Y-axis for this kind of control (see https://github.com/libretro/FBNeo/issues/28).

**Comment by PlagueWizard** [comment]
I'll test it in the morning and see if it works. This will make playing g-loc and galaxy force way easier.

Thanks for all the hard work, dink too.

**Comment by RobLoach** [comment]
@PlagueWizard Wanted to thank you for pointing out the lobby IPs being public. The repo was private, but then became public without my knowledge. It's private again and I realized you wouldn't get the comment I posted. So, here you go.

We want it to be easy for people to play netplay together through the lobby, but have to bring it down for now. What was there was put together as more of a proof of concept than anything. There will be a more substantial lobby in place in the next coming weeks.

Would love to get your thoughts on it on Discord! Thanks again for all the help.
